### PR TITLE
Fix ordering issue when adding and removing bar models

### PR DESCRIPTION
**Pull request by matthewcheok**
## How was it tested?
*How did you verify that this change accomplished what you expected? Add more detail as needed.*
- [x] Wrote automated tests
- [ ] Built and ran on the iOS simulator
- [ ] Built and ran on a device

## Pull request checklist
*All items in this checklist must be completed before a pull request will be reviewed.*

- [ ] Risky changes have been put behind a feature flag, e.g. `CollectionViewConfiguration`
- [x] Added a [`CHANGELOG.md` entry](https://keepachangelog.com/en/1.0.0/) in the "Unreleased" section for any library changes

### CODE REVIEW

File: Example/EpoxyExample/Data/BeloIpsum.swift [hunk 1]
```diff
@@ -97,7 +97,9 @@ final class BeloIpsum {
     "Shepherd’s hut",
   ]
 
-  private static func makeSentence(wordCount: Int, startingAt: Int) -> (String, Int) {
+  private static func makeSentence(wordCount: Int, startingAt: Int)
+    -> (String, Int)
+  {
```
Comment (matthewcheok) [inline Example/EpoxyExample/Data/BeloIpsum.swift:102]: Updated when i ran linting

Comment (thedrick) [reply Example/EpoxyExample/Data/BeloIpsum.swift:102]: hmm I thought Eric fixed this in #14 - did you rebase?

**Review by thedrick — APPROVED** [review]

**Comment by matthewcheok** [comment]
I'm not sure I understand what the error is from `lint-podspec` - any suggestions on what to do here?

**Review by erichoracek — APPROVED** [review]
Thanks, just a few suggestions

**Pull request merged**